src/locales/en/en.json

```diff
@@ -996,11 +996,13 @@
           "about": "About {{ type }}"
         },
         "attributes": {
+          "fullaccess": "Full Access",
           "label": "Attributes",
           "description": "Essential details shared by the issuer, which verify and describe key information in this credential.",
           "details": "Attribute details",
-          "issuee": "Issuee",
-          "issuancedate": "Issuance Date",
+          "issuee": "Issuer",
+          "issuancedate": "Issuance Date Time",
+          "blocksaid": "Attributes block SAID",
```

Comment:
What are these hardcoded strings? I can't find them in [the design](https://www.figma.com/design/XN6K76yt13zxCUHPh1Egr8/Identity-wallet?node-id=19797-65693&node-type=section&t=Ppy4jqEUdDUH9K8V-0).

Reply:
If it is not necessary, I will ignore it.

Reply:
The blocksaid can be ignored, I may have forgotten to mention this. So `a.d` is the hash of the attributes and it doesn't make sense to show it.

Also included is the issuance date and the issuee identifier (related identifier) - `a.dt` and `a.i`. Both of these fields are shown in the UI outside of the attributes section, so not sure if they should be duplicated.

Reply:
Done. Please take a look.

Reply:
Change LGTM